components/security.rst

```diff
@@ -38,6 +32,17 @@ be used separately:
     It brings many layers of authentication together, allowing the creation
     of complex authentication systems.
 
+You can install each of them separately in your project:
+
+.. code-block:: terminal
+
+    $ composer require symfony/security-core
+    $ composer require symfony/security-http
+    $ composer require symfony/security-csrf
```

Comment:
I would not list the CSRF one in here. It's a bit of a weird sub component, as it has nothing to do with the other 3. It's just "security" in the broad sense, not the "having a user/admin/member-only feature" that this security doc is about.

Reply:
Do you mean removing the explanation of `symfony/security-csrf` above too?

Reply:
Ooh nvm, didn't know what was in the collapsed diff and I confused this with the Security guide article. :+1: on merging as-is